packages/@aws-cdk/aws-glue/lib/security-configuration.ts

```diff
@@ -0,0 +1,239 @@
+import * as kms from '@aws-cdk/aws-kms';
+import * as cdk from '@aws-cdk/core';
+import * as constructs from 'constructs';
+import { CfnSecurityConfiguration } from './glue.generated';
+
+/**
+ * Interface representing a created or an imported {@link SecurityConfiguration}.
+ */
+export interface ISecurityConfiguration extends cdk.IResource {
+  /**
+   * The name of the security configuration.
+   * @attribute
+   */
+  readonly securityConfigurationName: string;
+}
+
+/**
+ * Attributes for importing {@link SecurityConfiguration}.
+ */
+export interface SecurityConfigurationAttributes {
+  /**
+   * The name of the security configuration.
+   */
+  readonly securityConfigurationName: string;
+}
+
+/**
+ * Encryption mode for S3.
+ */
+export enum S3EncryptionMode {
+  /**
+   * No encryption.
+   */
+  DISABLED = 'DISABLED',
+
+  /**
+   * Server side encryption (SSE) with an Amazon S3-managed key.
+   *
+   * @see https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html
+   */
+  S3_MANAGED = 'SSE-S3',
+
+  /**
+   * Server-side encryption (SSE) with an AWS KMS key managed by the account owner.
+   *
+   * @see https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingKMSEncryption.html
+   */
+  KMS = 'SSE-KMS',
+}
+
+/**
+ * Encryption mode for CloudWatch Logs.
+ */
+export enum CloudWatchEncryptionMode {
```

Comment:
Wondering where you got the allowed values for these enums, I don't see it in CloudFormation docs. 

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-securityconfiguration-cloudwatchencryption.html#cfn-glue-securityconfiguration-cloudwatchencryption-cloudwatchencryptionmode

Would be good to have a link here. Same for the other enums.

Reply:
I had to fallback to API docs. I added them in the docs above.